ui/src/overview.tsx

```diff
@@ -91,17 +90,31 @@ function AddCriteriaButton(props: { group: string | GroupKind }) {
     [props.group]
   );
 
-  const items = getCriteriaTitles();
+  // TODO(tjennison): Fetch configs from the backend.
+  const configs = [
+    {
+      type: "concept",
+      title: "Conditions",
+      defaultName: "Contains Conditions Codes",
+      entity: "condition",
```

Comment:
Is this supposed to be the same interface as CriteriaConfig? Why is `entity` not defined there?

Also - may be worth typing this array

Reply:
That comes from the concept plugin config. I've broken that out into a separate object to better indicate that.